jupyter-extension/cypress/integration/repo-datum-mode-switch.ts

```diff
@@ -1,10 +1,11 @@
 describe('switching between repo and datum mode', () => {
   beforeEach(() => {
-    cy.unmountAllRepos();
     cy.resetApp();
     cy.isAppReady();
+    cy.unmountAllRepos();
```

Comment:
This is copied over from the other test suite's `before` behavior. Didn't realize there were two suites with two `before` behaviors.